scripts/gulp/preset.ts

```diff
@@ -0,0 +1,34 @@
+import { task, parallel } from 'gulp';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as tsPaths from 'tsconfig-paths';
+
+function registerTsPaths() {
+  const { compilerOptions } = JSON.parse(
+    fs.readFileSync(path.resolve(__dirname, '../../tsconfig.base.v0.json'), 'utf-8'),
+  );
+  tsPaths.register({
+    baseUrl: __dirname,
+    paths: compilerOptions.paths,
+  });
+}
+export function preset() {
+  // add node_modules/.bin to the path so we can invoke .bin CLIs in tasks
+  process.env.PATH = process.env.PATH + path.delimiter + path.resolve(__dirname, '../..', 'node_modules', '.bin');
```

Comment:
q: is this still needed ?

Reply:
As we still have `scripts/gulp/plugins/gulp-doctoc.ts` and some other calls it's probably still needed.

Reply:
hmm, I will try remove it and check what happens 📟

Reply:
will do as follow-up